packages/solana/lib/src/rpc/dto/parsed_message/spl_token_transfer_checked_info.dart

```diff
@@ -22,4 +25,7 @@ class SplTokenTransferCheckedInfo {
   final TokenAmount tokenAmount;
   final String source;
   final String destination;
+  final String mint;
+  final String multisigAuthority;
+  final List<String> signers;
```

Comment:
Not sure if this fields are nullable, will try to investigate it.

Reply:
After some digging, this is what I understand:
1. `mint` is non-nullable
2. `multisigAuthority` is non-nullable
2. `signers` is nullable (maybe do empty list as a default)

Reference for `TransferChecked`.
https://docs.rs/solana-transaction-status/latest/src/solana_transaction_status/parse_token.rs.html#342

`mint` and `multisigAuthority` seems to be always included in the model.

However `signers` are included based on some condition. 
https://docs.rs/solana-transaction-status/latest/src/solana_transaction_status/parse_token.rs.html#477


Let me know if you see this same way?

Reply:
I agree that `mint` appears to be non-nullable.

But looking at `parse_signers` method, I think it provides **either** `multisig_field_name` (`"multisigAuthority"`) and `"signers"` **or** `owner_field_name` (`"authority"`).

Reply:
Thanks for looking into it.

After closer inspection, I do agree with your conclusion. `SplTokenTransferInfo` model is also affected by it it seems.

I'll update code with changes in a bit.

Reply:
Updated models + added references to Solana Rust models.